packages/backend/src/test/testApiServer.ts

```diff
@@ -0,0 +1,10 @@
+import Router from '@koa/router'
+import { Logger } from '@l2beat/common'
+import { agent } from 'supertest'
+
+import { ApiServer } from '../../src/api/ApiServer'
+
+export function createTestApiServer(routers: Router[]) {
```

Comment:
dont we have such helper somewhere already?

Reply:
I don't think so, that's search result on master 
<img width="368" alt="Screenshot 2022-12-21 at 15 36 12" src="https://user-images.githubusercontent.com/38423054/208930274-5b5801d2-e098-4b87-9479-52ec7ee33fc8.png">